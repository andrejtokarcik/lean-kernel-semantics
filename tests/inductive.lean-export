1 #NS 0 u
2 #NS 0 list
3 #NS 0 alpha
1 #UP 1
0 #ES 1
1 #EP #BD 3 0 0
4 #NS 2 nil
2 #EC 2 1
3 #EV 0
4 #EA 2 3
5 #EP #BD 3 0 4
5 #NS 2 cons
6 #NS 0 a
6 #EV 1
7 #EA 2 6
8 #EV 2
9 #EA 2 8
10 #EP #BD 6 7 9
11 #EP #BD 6 3 10
12 #EP #BI 3 0 11
#IND 1 2 1 2 4 5 5 12 1
