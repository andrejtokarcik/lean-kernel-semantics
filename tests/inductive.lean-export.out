<T>
  <k>
    .
  </k>
  <names>
    0 |-> *empty*
    1 |-> u
    2 |-> list
    3 |-> alpha
    4 |-> list.nil
    5 |-> list.cons
    6 |-> a
  </names>
  <universes>
    0 |-> 0
    1 |-> u
  </universes>
  <expressions>
    0 |-> u
    1 |-> Π (alpha : u) → u
    2 |-> list
    3 |-> #0
    4 |-> list (#0)
    5 |-> Π (alpha : u) → list (#0)
    6 |-> #1
    7 |-> list (#1)
    8 |-> #2
    9 |-> list (#2)
    10 |-> Π (a : list (#1)) → list (#2)
    11 |-> Π (a : #0) → Π (a : list (#1)) → list (#2)
    12 |-> Π {alpha : u} → Π (a : #0) → Π (a : list (#1)) → list (#2)
  </expressions>
  <def-types>
    list |-> Π (alpha : u) → u
    list.cons |-> Π {alpha : u} → Π (a : #0) → Π (a : list (#1)) → list (#2)
    list.nil |-> Π (alpha : u) → list (#0)
  </def-types>
  <def-values>
    .Map
  </def-values>
  <exit-code>
    0
  </exit-code>
</T>
